variables.tf

```diff
@@ -26,49 +26,55 @@ variable ssh_key_name {
 
 variable zone_id {
   description = "Route53 Zone ID"
-  default = ""
+  default     = ""
 }
 
 variable host_name {
   description = "DNS host name"
-  default = ""
-}
-
-variable quay_username {
-  description = "Quay username"
-  default = "dontspamus"
-}
-
-variable quay_password {
-  description = "Quay password"
-}
-
-variable init_script {
-  description = "bash code executed before `docker-compose up -d` is called, example: `\"${file(\"init.sh\")}\"`"
-  default = ""
+  default     = ""
 }
 
 variable in_open_ports {
-  type = "list"
+  type    = list
   default = []
 }
 
 variable in_cidr_blocks {
-  type = "list"
+  type    = list
   default = ["0.0.0.0/0"]
 }
 
 variable in_source_security_group {
   description = "Security group to receive SSH access"
-  type = "string"
-  default = ""
+  type        = string
+  default     = ""
 }
 
 variable disk_utilization_alarm_threshold {
-  description = "disk occupation alarm threshold (% of disk utilization)"
-  default = "80"
+  description = "Disk occupation alarm threshold (% of disk utilization), default 80.\n"
+  type        = number
+  default     = 80
+}
+
+variable sns_alarm_enabled {
+  description = "Determine if the cloudwatch alarm will be created or not"
+  type        = bool
+  default     = true
+}
+
+variable alarm_actions {
```

Comment:
Nit: sorry to bother but I am a stickler for names, wouldn't it be better to prepend `disk_utilization_alarm_` to these names to show that they pertain to the same context?

Reply:
I completely agree, I am not that good with names 😆

Reply:
Same for ok_actions?

Reply:
We could simply use the same actions both for alarm and ok state actions

Reply:
Perhaps you forgot to push a commit regarding renaming?

Reply:
Sorry, I didn't mean two variables, but the same variable for both alarm and OK actions

Reply:
I am using the same variable (`disk_utilization_alarm_actions`) for both alarm and ok actions.

Reply:
Aw crap, I must have looked at the wrong commit. 🤦